### DISCUSSION TITLE
Color Picker Hex code incompatible to web

### DISCUSSION THREAD

**Discussion by siraniks** [d2phap/ImageGlass#515] [General] [answered]
### System info:
- ImageGlass version: 6.0.12.29
- OS version: Windows 10 Pro 64bit

Color picker has 8 values than most applicable code is only 6
I can't use it on other application like Photoshop or web

and I think there's an extra "FF" on each code

**Comment by d2phap** [comment, marked as answer]
Hi @siraniks,

You can uncheck the option **Use HEX with alpha code** in Settings > Color Picker

**Comment by siraniks** [comment]
oh I thought it was a bug, thank you :)
